docs/configuration/thumbnails.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Thumbnails
+---
+Thumbnails
+==========
+
+Bolt comes with a thumbnail system which allows you to scale and crop your images.
+This is particularly useful if images need to fit into a specific layout. 
+The reduced file size can also increase the performance of your site.
+
+The thumbnail settings are defined in `config.yml` under the `thumbnails` key.
+Here is an example with the default settings:
+
+```yaml
+thumbnails:
+    default_thumbnail: [ 160, 120 ]
+    default_image: [ 1000, 750 ]
+    quality: 80
+    cropping: crop
+    notfound_image: view/img/default_notfound.png
+    error_image: view/img/default_error.png
+    save_files: false
+    allow_upscale: false
+    exif_orientation: true
+#    browser_cache_time: 2592000
```

Comment:
The service provider [passes that value](https://github.com/bolt/bolt-thumbs/blob/master/src/ServiceProvider.php#L31) into the [Responder](https://github.com/bolt/bolt-thumbs/blob/master/src/Responder.php#L56) class and then uses it as a parameter to the [caching call](https://github.com/bolt/bolt-thumbs/blob/master/src/Responder.php#L97).